### PR TITLE
Update paged query

### PR DESCRIPTION
**Pull request by bingyanglin**
Update paged query

### CODE REVIEW

File: src/storage.rs [hunk 1]
```diff
@@ -121,6 +123,48 @@ impl CQLSession {
         self.0.query(CREATE_EDGE_TABLE_QUERY)?;
         Ok(())
     }
+
+    fn paged_query(
+        &self,
+        query: String,
```
Comment (wusyong) [inline src/storage.rs]: Why is this String. According to original type, it's a trait bound with `ToString`. Could this just be `&str`?

File: src/storage.rs [hunk 2]
```diff
@@ -121,6 +123,48 @@ impl CQLSession {
...
+                .query_with_params(query.clone(), params.finalize())
+                .and_then(|frame| frame.get_body())?;
+
+            let metadata_res: error::Result<RowsMetadata> = body
+                .as_rows_metadata()
+                .ok_or("Pager query should yield a vector of rows".into());
+            let metadata = metadata_res?;
+
+            let has_more_pages =
+                Some(RowsMetadataFlag::has_has_more_pages(metadata.flags.clone())).unwrap();
```
Comment (wusyong) [inline src/storage.rs]: This unwrap is not needed. Please use other methods or `if let` to match.

File: src/storage.rs [hunk 3]
```diff
@@ -121,6 +123,48 @@ impl CQLSession {
...
+            let new_rows = body.into_rows().unwrap();
+            rows = [&rows[..], &new_rows[..]].concat();
```
Comment (wusyong) [inline src/storage.rs]: Just use append instead.

File: src/storage.rs (hunk 1 above)
Comment (bingyanglin) [reply src/storage.rs]: Modified. Thank you for pointing this out.

File: src/storage.rs (hunk 2 above)
Comment (bingyanglin) [reply src/storage.rs]: Modified. Thank you for pointing this out.

File: src/storage.rs (hunk 3 above)
Comment (bingyanglin) [reply src/storage.rs]: Modified. Thank you.

**Pull request merged**